doc/source/introduction_intro.rst

```diff
@@ -14,49 +14,93 @@ to facilitate access to and exchange of genomics data across remote sites.
+For more details on web APIs, see `this wikipedia page <https://en.wikipedia.org/wiki/Web_API>`_  
 
-The schemas are written in Avro Interactive Data Language (extension .avdl). 
+HTTP protocol
+.............
+API clients use HTTP (`Hypertext Transfer Protocol <https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol>`_) for sending API queries and getting responses.
+HTTP is the same protocol that web browsers use for requesting pages from web servers.
 
-:ref:`avro` is a data serialization system, it defines how data gets transported across the internet.
-Here, data means both the request ('send me RNASeq for gene X in sample Y') and the response (the BAM file). Apache Avro only sends data in
-:ref:`json` or Binary Avro format, so requests and responses must be converted into one of those formats.
+In a typical conversation, a client sends a GET request to a server to ask for a specific resource, such as a set of reads.
+If the request contains appropriate identification and authorization information, the server responds with a copy of the requested information.
 
-For more details on AVRO, see :ref:`avro`
+Many GA4GH API servers will also support HTTPS, a variant of HTTP that uses encryption to protect messages from eavesdropping and modification while in transit.
 
-For more details on JSON, see :ref:`json`
+ReST (Representational State Transfer) conventions
+..................................................
+The GA4GH APIs follow `ReST conventions <https://en.wikipedia.org/wiki/Representational_state_transfer>`_ for web services on HTTP.  In particular:
 
+* The API defines many different kinds of resources, some of which are collections of other kinds of resources.
+* Each resource is identified by a unique URI.
+* Clients can create resources and retrieve information about them by applying standard HTTP verbs like GET, PUT, POST, DELETE to appropriate URIs.
+* GET is used to list collections and retrieve individual items. GET requests never have side effects.
+* PUT is used to replace collections and items.  PUT requests are idempotent - repeating the same PUT request is equivalent to doing it just once.
+* POST is used to modify collections.
+* DELETE is used to remove collections and items.  DELETE requests are idempotent - repeating the same DELETE request is equivalent to doing it just once.
+
+TODO examples of GA4GH resources and their use.
+
+Paging
+......
+
+All GA4GH APIs implement a paging convention, so that the amount of information returned in a single response can be bounded. All
+GA4GH APIs have paging support, even those that are very unlikely to return large amounts of information. This convention helps avoid breaking changes later:
+paging often becomes necessary eventually, and adding paging in later would break clients that don't expect it.
+
+TODO more details of paging convention and use.
+
```

Comment:
I'm happy to flesh this out more in a follow-up PR, if you'd prefer that Will. I think this is a great start.